app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1499,3 +1519,293 @@ $include-html-global-classes: $include-html-classes;
+.section {
+//  border: 5px solid red;
+}
+
+.section ul li {
+  border: 5px solid red;
+  background: black;
+  display: inline-block;
+}
+
+
+.top-bar {
+//  border: 5px solid red; 
+}
+
+.top-bar a {
+  margin: 0px 20px;
+}
+
+.top-bar ul li {
+  background-color: #5c5c5c;
+}
+
+.right {
+  background-color: #2a2a2a;
+}
+
+.left {
+  background-color: #2a2a2a;
+}
+
+.padder {
+  height: 500px;
+  background: url("feast_hall_dressed_table_.jpg");
+  background-size: cover;
+  background-attachment: fixed;
+}
+
+.ta{
+  background-color: #70857E;
+}
+
+.filter {
+  margin-top: 50px;
+  color: #FFFFFF;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 72px;
+//  border: 5px solid red;
+}
+
+.dropdown {
+  margin-bottom: 50px;
+}
+
+.row-items {
+  max-width: 1000px;
+}
+
+.items {
+  border: 1px solid red;
+  margin: 10px auto;
+}
+
+.item-image {
+  border: 1px solid red;
+  text-align: center;
+}
+
+.item-show-image {
+  border: 1px solid red;
+  text-align: center;
+}
+
+.item-title {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 58px;
+}
+
+.item-title a {
+  color: white;
+  font-size: 24px;
+}
+
+.item-price {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 24px;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.item-show-price {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 52px;
+  margin: 20px 0 10px 0;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.item-description {
+  border: 1px solid red;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 24px;
+  margin-top: 10px;
+  padding: 0px 5px;
+  background: black;
+  opacity: 0.4;
+}
+
+.item-show-description {
+  border: 1px solid red;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 48px;
+  margin: 10px 0;
+  padding: 0px 5px;
+}
+
+.item-categories {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 40px;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.back-link {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 40px;
+  margin: 10px auto;
+  font-family: 'Cardinal', Arial, sans-serif;
+}
+
+.add-to-cart {
+  border: 1px solid red;
+  text-align: center;
+  padding-top: 20px;
+  background: black;
+  opacity: 0.4;
+}
+
+.large-3.columns {
+  padding-top: 10px;
+  background: #5c5c5c;
+  border: 1px solid red;
+}
+
+.login-title {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'Cardinal', Arial, sans-serif;
+  font-size: 58px;
+  color: white;
+}
+
+.form {
+  border: 1px solid red;
+  margin-top: 30px;
+  color: white;
+}
+
+.cart h1 {
+  border: 1px solid red;
+  text-align: center;
+  margin-top: 50px;
+  font-family: 'SketchGothic', Arial, sans-serif;
+  font-size: 58px;
+  color: white;
+}
+
+.cart-item-total {
+  border: 1px solid red;
+  text-align: center;
+  font-size: 24px;
+  margin: 20px 0 10px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `20px 0 10px` instead of `20px 0 10px 0`